docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,12 @@ develop
     *    - Type
          - Change
 
+    *    - |fixed| |userbreak|
+         - Cassandra KVS `getMetadataForTables` method now does not contain entries for tables that do not exist in Cassandra.
+           Previously, when a table was dropped, an empty byte array would be written into the _metadata table to mark it as deleted.
+           Furthermore, the table reference keys of the returned map have capitalisation matching the table names in Cassandra.
```

Comment:
So one possible concern about this change is that now when we drop a table we need to call `deleteAllMetadataRows`, which triggers a range scan on _metadata across the ASCII range from its upper to lower versions. Should we call this out here, along with 'please contact the AtlasDB team if this is an issue' or something like that?

Reply:
Makes sense. I don't think there should be cases where this comes back to bite us, but makes sense to call it out.